recipes/nasm/all/conanfile.py

```diff
@@ -64,12 +62,11 @@ def generate(self):
 
     def build(self):
         apply_conandata_patches(self)
-
+        autotools = Autotools(self)
         if is_msvc(self):
             with chdir(self, self.source_folder):
-                self.run("nmake /f {}".format(os.path.join("Mkfiles", "msvc.mak")))
+                autotools.make(f'/f {os.path.join("Mkfiles", "msvc.mak")}')
```

Comment:
doesn't make sense

Reply:
Hi @SpaceIm,

Thanks for the review comments. The code for Autotools makes explicit mention of nmake so I have been assuming that it is designed to handle it: See Conan2.0 source file conan/tools/gnu/autotools.py:

>         if "-j" not in str_args and "nmake" not in make_program.lower():

Reply:
Hi @SpaceIm,
Okay, restored use of self.run as @prince-chrismc did not like using tools.gnu:make_program to specify nmake from within a recipe.